docs/spec.yml

```diff
@@ -250,11 +250,11 @@ paths:
           description: Unprocessable request
         '500':
           description: Validation Error
-  '/v3/technical-records{systemNumber}/{createdTimestamp}':
```

Comment:
looks like you have deleted an API, instead of adding a new one. Is that intentional?